src/universal/utils/fromNow.js

```diff
@@ -0,0 +1,28 @@
+const thresholds = {
```

Comment:
I forgot to tell you how much I liked this @mattkrick . Really compact. To the point. I think you should publish this as as gist then tweet it!